collectors/carbonblack/collector.js

```diff
@@ -0,0 +1,160 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2020, Alert Logic, Inc
+ * @doc
+ *
+ * carbonblack class.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+'use strict';
+
+const moment = require('moment');
+const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
+const parse = require('@alertlogic/al-collector-js').Parse;
+const utils = require("./utils");
+
+
+let typeIdPaths = [];
+
+let tsPaths = [];
```

Comment:
I suggest pulling message timestamp from the eventTime field in the message returned by the CarbonBlack API
```
        {
            "requestUrl": null,
            "eventTime": 1529332687006,
            "eventId": "37075c01730511e89504c9ba022c3fbf",
            "loginName": "bs@carbonblack.com",
            "orgName": "example.org",
            "flagged": false,
            "clientIp": "192.0.2.3",
            "verbose": false,
            "description": "Logged in successfully"
        }
```

Reply:
Ignore, I see that different fields are pulled in utils module depending on the API being called